src/foam/nanos/auth/Address.js

```diff
@@ -28,15 +28,26 @@ foam.CLASS({
       class: 'Boolean',
       name: 'deleted'
     },
+    {
+      class: 'Boolean',
+      name: 'structured',
+      value: true
+    },
     {
       class: 'Int',
       name: 'buildingNumber',
       documentation: 'Building number'
     },
     {
       class: 'String',
-      name: 'address',
-      required: true
+      name: 'address1',
+      //required: true
+      displayWidth: 50
+    },
+    {
+      class: 'String',
+      name: 'address2',
+      displayWidth: 50
```

Comment:
What are address1 & address2 meant to be used for? Can you add some documentation?

Reply:
@minsunny Document how the structured field affects the various other address fields.